.gitignore

```diff
@@ -0,0 +1,5 @@
+venv/
+.env
+__pycache__
+db.sqlite3
+.idea
```

Comment:
I still see those files

Reply:
`git rm -r .idea`
`git add .`
`git commit -m ‘idea was removed’`
`git push origin <…>`
Make sure you did all steps